config/v1/types_network.go

```diff
@@ -85,6 +85,16 @@ type NetworkStatus struct {
 
 	// Migration contains the cluster network migration configuration.
 	Migration *NetworkMigration `json:"migration,omitempty"`
+
+	// conditions represents the observations of a network.config current state.
+	// Known .status.conditions.type are: "NetworkTypeMigrationInProgress", "NetworkTypeMigrationMTUReady",
+	// "NetworkTypeMigrationTargetCNIAvailable", "NetworkTypeMigrationTargetCNIInUse"
+	// and "NetworkTypeMigrationOriginalCNIPurged"
+	// +optional
+	// +listType=map
+	// +listMapKey=type
+	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
```

Comment:
```suggestion
	// +optional
	// +listType=map
	// +listMapKey=type
	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
	// +patchMergeKey=type
	// +patchStrategy=merge
	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
```
Please include the patch merge to be consistent with other `Conditions` fields we have on other CRDs